package.json

```diff
@@ -126,6 +126,7 @@
             "ahk": "autohotkey",
             "autoit": "autoit3",
             "kotlin": "cd $dir && kotlinc $fileName -include-runtime -d $fileNameWithoutExt.jar && java -jar $fileNameWithoutExt.jar",
+            "d": "cd $dir && dmd $fileName && $dir$fileNameWithoutExt",
```

Comment:
You also need to add ".d" definition in "code-runner.executorMapByFileExtension", since the `d` is not a default language id provided by VS Code.